go.mod

```diff
@@ -344,7 +344,8 @@ require (
 )
 
 replace (
-	github.com/CosmWasm/wasmd => github.com/sei-protocol/sei-wasmd v0.2.9
+	github.com/CosmWasm/wasmd => github.com/sei-protocol/sei-wasmd v0.0.0-20241203200623-bdd858fb6f9b
```

Comment:
We should cut a tag instead of using a hash here

Reply:
updated